### PR TITLE
removed polymer dep

### PR DESCRIPTION
**Pull request by e111077**

### CODE REVIEW

**Comment by notwaldorf** [comment]
@cdata ping! I think `webcomponents.js` would benefit from this :)

File: promise-polyfill-lite.html [hunk 1]
```diff
@@ -7,10 +7,16 @@
 Code distributed by Google as part of the polymer project is also
 subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
 -->
-<link rel="import" href="../polymer/polymer.html">
 <script src='./Promise.js'></script>
 <script>
 if (!window.Promise) {
-  window.Promise = MakePromise(Polymer.Base.async);
+  var asap;
+  if (Polymer) {
```
Comment (notwaldorf) [inline promise-polyfill-lite.html]: This is technically still a dependency on Polymer, no? Can we just replace it with what `webcomponents.js` did? https://github.com/webcomponents/webcomponentsjs/blob/v1/src/WebComponents/Promise.js#L130

**Comment by e111077** [comment]
Sounds like were essentially removing this polyfill altogether and replace it with that one. I'm A-okay with that.

**Comment by notwaldorf** [comment]
@e111077 Well uhhhhh that polyfill needs to live somewhere that isn't webcomponents.js, and i thought maybe it should be here?

@azakus @sorvell @kevinpschaaf if they have a strong opinion

File: promise-polyfill-lite.html (hunk 1 above)
Comment (valdrinkoshi) [reply promise-polyfill-lite.html]: should this be `window.Polymer`?

Comment (e111077) [reply promise-polyfill-lite.html]: 👍

File: promise-polyfill-lite.html [hunk 2]
```diff
@@ -7,10 +7,16 @@
 Code distributed by Google as part of the polymer project is also
 subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
 -->
-<link rel="import" href="../polymer/polymer.html">
 <script src='./Promise.js'></script>
 <script>
 if (!window.Promise) {
-  window.Promise = MakePromise(Polymer.Base.async);
+  var asap;
```
Comment (cdata) [inline promise-polyfill-lite.html]: This will create a global `asap` variable. Maybe wrap in an IIFE?

**Review by cdata — APPROVED** [review]
`setTimeout` is not a satisfying alternative to microtask timing. That said, I don't think it's a huge deal. YOLO?

One suggestion, but otherwise LGTM.

**Pull request merged**